includes/admin/class-wcmp-export.php

```diff
@@ -980,7 +980,7 @@ public function getShipmentData(array $ids, WC_Order $order): array
      *
      * @return float
      */
-    public static function get_item_weight_kg($item, WC_Order $order): float
+    public static function getItemWeight_kg($item, WC_Order $order): float
```

Comment:
Method naam ziet er gek uit als Camel Case en Snake Case tegelijk gebruikt. Mijn voorkeur is Camel Case

Reply:
changed